schema-registry/common/src/test/resources/avro/nested/nested-simple.avsc

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Could you move this file to /resources/avro/composites/ ?

Reply:
Ditto above.